crates/astria-composer/src/executor/mod.rs

```diff
@@ -422,9 +424,9 @@ impl Executor {
 async fn get_latest_nonce(
     client: sequencer_client::HttpClient,
     address: Address,
+    metrics: &Metrics,
 ) -> eyre::Result<u32> {
     debug!("fetching latest nonce from sequencer");
-    metrics::counter!(crate::metrics_init::NONCE_FETCH_COUNT).increment(1);
```

Comment:
I moved this to inside `tryhard::retry_fn` on line 457, since it seems from the description ("The number of times we have attempted to fetch the nonce") like we want to count attempts rather than successes.

If we do mean success counts, I can revert this change and we could consider renaming the metric and description to reflect that?

Reply:
I think the original idea was not to count the individual GET requests but to count the "process" of getting a nonce.

Having said that, this metric feels a tad useless because it either closely tracks the number of failures/attempts (as it's doing now), or the number of data points added to the histogram.

I am actually leaning toward removing it altogether...